pkg/addons/data/DaemonSet.apps/openshift-azure-monitoring/etcd-metrics.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   labels:
+    azure.openshift.io/sync-pod-wait-for-readiness: "false"
```

Comment:
etcd-metrics is not depending on monitoring operator. so I think we can check for it. Or there is other reasons we doing this?

Reply:
Technically we could check it, but the line I'm suggesting is: customer facing => check it, not => don't